src/platform.config.ts

```diff
@@ -1,23 +1,22 @@
 import * as Pieces from "@pieces.app/pieces-os-client";
+import { PiecesClient } from "pieces-copilot-sdk";
 
 let BASE_URL: string;
 let WS_URL: string;
 
-if (Pieces.PlatformEnum.Linux) {
-  BASE_URL = 'http://localhost:5323';
-  WS_URL = 'ws://localhost:5323';
-}
-if(Pieces.PlatformEnum.Macos  || Pieces.PlatformEnum.Windows) {
-  BASE_URL = 'http://localhost:1000';
-  WS_URL = 'ws://localhost:1000';
-}
-else{
-  BASE_URL = 'http://localhost:1000';
-  WS_URL = 'ws://localhost:1000';
+// Define your port here 
+// LINUX PORT=5323
+// For Windows and MacOs PORT=1000
 
-}
+const PORT = 5323
```

Comment:
This part should be Dynamic. You can probably use the os package to check the os and change the port dynamically

Reply:
Sure, will try to test with "os" package and its implementation